contracts/VestedPayment.sol

```diff
@@ -0,0 +1,85 @@
+pragma solidity ^0.4.13;
+
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./LifToken.sol";
+
+contract VestedPayment is Ownable {
+  using SafeMath for uint256;
+
+  // when the vested schedule starts
+  uint256 public startTime;
+
+  // how much each period will last
+  uint256 public timePerPeriod;
+
+  // how much periods will have in total
+  uint256 public totalPeriods;
+
+  // the amount of tokens to be vested in total
+  uint256 public tokens;
+
+  // how much tokens were claimed
+  uint256 public claimed;
+
+  // the token contract
+  LifToken public token;
+
+  // how much cliff period teh vesting schedule will have
```

Comment:
I'd say it this way: "duration (in periods) of the initial cliff in the vesting schedule"

Reply:
and the variable could be `cliffDuration`

Reply:
:+1: